core/src/test/java/tachyon/master/LocalTachyonCluster.java

```diff
@@ -144,10 +144,11 @@ private void mkdir(String path) throws IOException {
   }
 
   public void start() throws IOException {
-    mTachyonHome =
-        File.createTempFile("Tachyon", "").getAbsoluteFile() + "U" + System.currentTimeMillis();
+    mTachyonHome = File.createTempFile("Tachyon",
+        "U" + System.currentTimeMillis()).getAbsolutePath();
```

Comment:
I don't know why the previous style need dir name to be tmpDir.name + "U" + System.currentTimeMillis(), maybe the time is more convenient to debug? I just follow the code style.
If you want it to be cleaner, the following 2 approach can work, too.
1. mTachyonHome = File.createTempFile("Tachyon", "").getAbsolutePath()
2. mTachyonHome = "/tmp/Tachyon" + System.currentTimeMillis();